.azure-pipelines/unit_test.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+trigger:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+strategy:
+  matrix:
+    Python36:
+      python.version: '3.6'
+    Python38:
+      python.version: '3.8'
+
+steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python $(python.version)'
+    inputs:
+      versionSpec: '$(python.version)'
+
+  - script: |
+      python3 -m pip install --upgrade flake8 prometheus-client opencensus opencensus-ext-azure
+    displayName: Install External Packages
+  - script: |
+      python3 -m flake8
+    displayName: Run Lint Tests
```

Comment:
So, maybe just an observation. right now I do not believe all python files are formatted correctly w/ respect to lint. So is it the case that the first time we run the pipeline it will fail on all these files?

Reply:
> I do not believe all python files are formatted correctly w/ respect to lint.
Yeah, you are right. I will submit another PR to handle them.
In another hand, unit-test pass rate is 64.5% in my box, we can fix them later.
So in current stage, lint test and unit test are non-mandatory with `continueOnError` param, which will not affect PR checkin.

Reply:
OK cool